recipes/freeimage/all/conandata.yml

```diff
@@ -1,6 +1,6 @@
 sources:
   "3.18.0":
-    url: "http://downloads.sourceforge.net/project/freeimage/Source Distribution/3.18.0/FreeImage3180.zip"
+    url: "http://downloads.sourceforge.net/project/freeimage/Source%20Distribution/3.18.0/FreeImage3180.zip"
```

Comment:
Thanks @luzpaz - turns out `+` did not work, but `%20` does

Reply:
Thanks!